src/Features/LanguageServer/Protocol/Handler/SemanticTokens/SemanticTokensHelpers.cs

```diff
@@ -162,6 +162,41 @@ static SemanticTokensHelpers()
             return (ComputeTokens(text.Lines, updatedClassifiedSpans, tokenTypesToIndex), isFinalized);
         }
 
+        private static async Task GetClassifiedSpansForDocumentAsync(
+            Document document,
+            TextSpan textSpan,
+            ArrayBuilder<ClassifiedSpan> classifiedSpans,
+            bool isRazorDoc,
```

Comment:
more of a naming nit - perhaps we could rename this to be something like `includeSyntacticClassifications`